doc/concepts/configuration.rst

```diff
@@ -171,55 +178,128 @@ You can learn more about configuring replication from :ref:`Replication tutorial
 
 
 .. _configuration_application:
+.. _configuration_application_roles:
+
+Enabling and configuring roles
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+An application role is a Lua module that implements specific functions or logic.
+You can enable or turn off a particular role for certain instances in a configuration without restarting these instances.
 
-Loading an application
-**********************
+This section describes how to enable and configure roles.
+To learn how to develop custom roles, see :ref:`application_roles`.
```

Comment:
You write about custom roles, but there there is no explanation of _non custom_ roles. Maybe add a sentence about them to the first paragraph?